spec/cli_spec.rb

```diff
@@ -15,12 +15,22 @@
     expect(err).to match(/input file \/nonexistent( is)? missing/)
   end
 
-  it 'converts and validates sample book' do
+  it 'converts sample book to epub and validates it' do
     infile = example_file 'sample-book.adoc'
     outfile = temp_file 'sample-book.epub'
 
     _, _, res = run_command asciidoctor_epub3_bin, '-a', 'ebook-validate', infile, '-o', outfile
     expect(res.exitstatus).to eq(0)
     expect(File).to exist(outfile)
   end
+
+  # TODO: enable on MacOS when https://github.com/asciidoctor/asciidoctor-epub3/issues/236 is fixed
+  it 'converts sample book to mobi', :skip => !os_darwin? do
```

Comment:
I tend to just use `unless os_darwin?` after `end`.

If you were to use the hash arguments, it should be:

```
it 'converts sample book to mobi', skip: !os_darwin? do
```

The `:skip => !os_darwin?` is the old Hash syntax.

Reply:
For some reason, the `skip: !os_darwin?` didn't actually skip it, so I thought I'm passing it the wrong way. 

> I tend to just use `unless os_darwin?` after end.

If I understand how things work properly, this way rspec will have no idea about test existence and will not report it as skipped. Correct?

Reply:
Okay, I inverted logic accidentally :) I want to skip it _on_ MacOS.

Reply:
There you go!